src/components/search/headerSearchButton.js

```diff
@@ -5,12 +5,16 @@ const HeaderSearchButton = () => {
   const { toggleSearchOverlay } = useContext(SearchOverlayOpenContext);
 
   return (
-    <button className="ood-header__search-button" onClick={toggleSearchOverlay}>
+    <button
+      type="button"
+      className="ood-header__search-button"
+      onClick={toggleSearchOverlay}
+    >
```

Comment:
I meant to do this in another PR but accidentally include it here. It improves things so might as well stay here.